src/main/java/com/google/devtools/build/lib/runtime/BuildSummaryStatsModule.java

```diff
@@ -60,6 +63,11 @@ public class BuildSummaryStatsModule extends BlazeModule {
   private long executionEndMillis;
   private SpawnStats spawnStats;
   private Path profilePath;
+  private static final long UNKNOWN_CPU_TIME = -1;
+  //If anyone of the CPU time below is UNKNOWN, then the total CPU time become UNKNOWN.
```

Comment:
I think, that this comment make more sense as documentation of method `sumCpuTimes `. Please move it there.